vehicles/Steering/gimbal.cc

```diff
@@ -507,28 +516,32 @@ void selfTestTask(void* arg) {
   OLED->ShowString(0, 5, (uint8_t*)"GY");
   OLED->ShowString(1, 0, (uint8_t*)"SL");
   OLED->ShowString(1, 5, (uint8_t*)"SR");
-  OLED->ShowString(1, 10, (uint8_t*)"LD");
-  //  OLED->ShowString(2, 0, (uint8_t*)"FL");
-  //  OLED->ShowString(2, 5, (uint8_t*)"FR");
-  //  OLED->ShowString(2, 10, (uint8_t*)"BL");
-  //  OLED->ShowString(2, 15, (uint8_t*)"BR");
-  OLED->ShowString(3, 0, (uint8_t*)"Cali");
-  OLED->ShowString(3, 7, (uint8_t*)"Temp:");
+  OLED->ShowString(2, 0, (uint8_t*)"LD");
+  OLED->ShowString(2, 5, (uint8_t*)"Ldr");
+  OLED->ShowString(3, 0, (uint8_t*)"Cal");
+  OLED->ShowString(3, 6, (uint8_t*)"Dbs");
+  OLED->ShowString(4, 0, (uint8_t*)"Temp:");
   //  OLED->ShowString(4, 0, (uint8_t*)"Ref");
-  OLED->ShowString(4, 6, (uint8_t*)"Dbus");
-  OLED->ShowString(4, 13, (uint8_t*)"Lidar");
+
+  OLED->ShowString(0, 15, (uint8_t*)"S");
+  OLED->ShowString(0, 18, (uint8_t*)"W");
```

Comment:
What are these 2? Steering and Wheels?

Reply:
Yes, the flag checkboxes now look like a table of position and motor type